features/polyExplorer/src/constants/index.js

```diff
@@ -20,21 +20,3 @@ export const INTRO_ANIMATIONS = {
     FORWARD: "forward",
```

Comment:
`INTRO_ANIMATIONS` can be removed too. It was used for the bullet points in `summary.jsx`.